app/main.js

```diff
@@ -7,6 +7,7 @@ import options from './setup/options';
 import components from './setup/components';
 
 import './setup/plugins';
+import urlFunctions from './mixins/url-functions';
```

Comment:
- [ ] I think it would be better if we add `helpers` or similar instead of `mixin`. The reason is, that you can never use `this` the way you use the code it here. The mixin structure could mislead other developers to add such logic in the future, which would crash it in old uses. Let us discuss this in person.

Reply:
@patric-eberle Ok let me know when you have time.